app/css/main.css

```diff
@@ -55,17 +94,18 @@ a {
 
 article p {
   font-family: 'Roboto', 'Noto', sans-serif;
-  font-size: 16px;
-  line-height: 26px;
+  font-size: 18px;
+  line-height: 28px;
+  max-width: 720px;
```

Comment:
Do you want the changes here?  We want main.css to be identical across sites. This change will affect articles in individual product sites, e.g. https://pwa-starter-kit.polymer-project.org/setup/

If styles are specific to base-blog.html then they should be added there.

Reply:
Yes, we need to set max-width for all paragraph. Ideal optimal line length for body text is around 50-60 characters, above 75 gets harder to read. Right now the paragraphs for individual doc site is too wide.